packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.tsx

```diff
@@ -45,8 +45,17 @@ export const PolygonAngle = ({
     const b = vec.dist(centerPoint, endPoints[1]);
     const c = vec.dist(endPoints[0], endPoints[1]);
 
+    let equation = (a ** 2 + b ** 2 - c ** 2) / (2 * a * b);
```

Comment:
Super duper tiny nit: rename `equation` to `expression` since there is no equal sign inside it.

I think if you want to get super technical, you can call it `lawOfCosinesRadicand`?? I tried looking it up and it doesn't seem to have a specific name 🤔